app/models/supplejack_api/site_activity.rb

```diff
@@ -38,7 +38,7 @@ def self.generate_activity(time = Time.now)
       site_activity.source_clicks = SupplejackApi::SourceActivity.get_source_clicks || 0
       SupplejackApi::SourceActivity.reset
       site_activity.calculate_total
-      site_activity.save
+      site_activity.save!
```

Comment:
the `!` bang is used for error handling?

Reply:
`!` will throw an exception if the `save` method fails